translations/translation_hu.xml

```diff
@@ -502,63 +501,63 @@ Ezt követően mindig változtathatsz más értékekkel és generálhatsz útvon
 A kísérletezés és különböző beállítások hatásainak megfigyelése különböző mezőkön a legjobb módja, hogy hozzászokj ehhez.
 "/>
     <text name="CP_help_page_courseGeneratorBaseFunctions_text" text="
-Basics:
-The first few values you see here, are real basic stuff.
-- Work width: Most tools get the exact work width detected and you won't have to care about this setting at all. Adjusting it, will have an impact on your overall course.
-- Multiple Tools: This setting is used when you want more then just one vehicle to work on your course. As this gets a bit more complicated, there is a separate help topic for it.
-- Number of Headlands: The best way to keep your vehicles on the field while turning at the end of each row is to add headlands.
-The Number of the Headlands multiplied by the work width should be at least the total length of your vehicle plus the attached tool.
-Center options:
-- Field center: There are different modes on how your field center pattern should look like. The classic and most used one is up/down.
-Spiral, Racetrack and Lands have their own specific advantages over the others. Lands for example will have the combine's pipe point out of the fruit most of the time to easier unload them.
-- Up/Down row direction: automatic mostly finds the best direction, but sometimes the longest edge fits better. If you are not happy with either one, choose manual and set the direction manually.
-- Rows to skip: This is a very helpful option to accelerate your work, as tools won't have to back up to turn into the next row.
-- Rows per Land: This has only impact when the field center is set to lands and will tell the generator how many rows each land should have. The more rows, the fewer lands will be generated.
-- Up/down row angle: When the Up/Down row direction is set to manual, this setting will tell the generate the direction.
+Alapok:
+Az első néhány érték, amit itt látsz, igazán alapvető dolog.
+- Munkaszélesség: A legtöbb gép munkaszélessége automatikusan beolvasásra kerül, így ezzel a beállítással egyáltalán nem kell törődnöd. Ennek módosítása hatással lesz a generált útvonalra.
+- Több eszköz: Ez a beállítás akkor használható, ha egynél több gépet akarsz egyidejűleg dolgoztatni egy útvonalon. Mivel ez egy kicsit bonyolultabb, van hozzá egy külön segítő rovat.
+- Fordulósávok száma: A legjobb módja annak, hogy a gépeidet a mezőn tartsd, miközben a sorok végén megfordulnak az, ha fordulósávokat adsz az útvonalhoz.
```

Comment:
Már akartam a múltkor is kérdezni, hogy a "mező" tényleg a jó fordítás-e a "field"-re. Én nem vagyok járatos a szakmában, de ha például mezőgazdasági GPS rendszerek magyar leírását, meg katalógusokat nézek, ott mindenhol a táblát használják.